cmd/flags/flags.go

```diff
@@ -39,6 +39,7 @@ const (
 	SyncTargetBlock      = "sync.targetBlock"
 	SyncGetMissingBlocks = "sync.getMissingBlocks"
```

Comment:
`SyncGetMissingBlocks` looks like no one else is using, can you help to remove? XDD

Reply:
please ignore this comment and rebase. XD

Reply:
XD